File: app/src/main/java/com/ceandt/moviedb/model/Cast.java
package com.ceandt.moviedb.model;

import com.google.gson.annotations.SerializedName;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class Cast {

    @SerializedName("cast_id")
    private Integer castId;
    @SerializedName("character")
    private String character;
    @SerializedName("credit_id")
    private String creditId;
    @SerializedName("gender")
    private Integer gender;
    @SerializedName("id")
    private Integer id;
    @SerializedName("name")
    private String name;
    @SerializedName("order")
    private Integer order;
    @SerializedName("profile_path")
    private String profilePath;

}

File: app/src/main/java/com/brunodiegom/movies/model/Cast.kt
package com.brunodiegom.movies.model

import com.google.gson.annotations.SerializedName

/**
 * Data model of [Cast] returned from server.
 */
data class Cast(
    @SerializedName("cast_id")
    val id: Int,
    @SerializedName("name")
    val name: String,
    @SerializedName("character")
    val character: String,
    @SerializedName("profile_path")
    val picture: String
)
